mongodb-advanced/components/people-component.js

```diff
@@ -0,0 +1,30 @@
+/*!
+ * Copyright (c) 2016-2017 Digital Bazaar, Inc. All rights reserved.
+ */
+export default {
+  controller: Ctrl,
+  templateUrl: 'mongodb-advanced/people-component.html'
+};
+
+function Ctrl($scope, exPeopleService) {
+  const self = this;
+  self.showMyModal = false;
+
+  exPeopleService.getPeople(people => {
+    self.savedPeople = people;
+  });
+
+  self.modalClosed = () => {
+    exPeopleService.getPeople(people => {
+      self.savedPeople = people;
+    });
+  };
+
+  self.deletePerson = name => {
+    exPeopleService.removePerson(name, () => {
```

Comment:
I just noticed how you've setup callbacks here and in the `people-service`.  In the front end promises are used.  So you don't pass callbacks to the service here.  If you're going to use the return from the service it would look like this:
```
exPeopleService.getPeople().then(people => self.savedPeople = peopple);
```